CI.md

```diff
@@ -0,0 +1,37 @@
+# Commands to trigger ci pipeline
+
+## Guide
+
+1. ci pipeline will be triggered when your comment on pull request matched trigger phrase.
+
+2. Some ci pipelines only response to people belong to specific organization like pingcap & tikv.
+
+3. "**Only use trigger phrase for build triggering**". What does that mean?
+
+   * Yes，this ci will be triggered only when your comment on pr matched trigger phrase.
+
+   * No，this ci will be triggered by every new commit on current pr，comment matched trigger phrase also trigger ci pipeline.
+
+## Commands
+
+| ci pipeline                              | Command                                                      | Only use trigger phrase for build triggering |
+| ---------------------------------------- | ------------------------------------------------------------ | -------------------------------------------- |
+| tidb_ghpr_build                          | .\*\/(merge\|run(-all-tests\|-build).\*)                     | No                                           |
```

Comment:
why use regular expression 😂 

it is not so different from listing out all commands: 

> /merge<br>/run-all-tests<br>/run-build

but the regexp here needs more effort to comprehend.

Reply:
Emm.  Listing out all commands is more simple and clear. I will correct them later.